src/app/urls.py

```diff
@@ -23,6 +23,7 @@
 urlpatterns = [
     path('', include('authentification.urls')),
     path('', include('server.urls')),
+    path('app/', include('piaf.urls')),
```

Comment:
A new app is created in order to isolate what is specific to our needs.
In the longer term we should be able to get rid of the _server_ app and use only a tiny part of the _api_ app while keeping this new app tiny.

Reply:
Good idea. This way we'll have both of the folder available to reproduce the required functionalities